Lib/test/test_ssl.py

```diff
@@ -480,10 +480,10 @@ def fail(cert, hostname):
         fail(cert, 'Xa.com')
         fail(cert, '.a.com')
 
-        # only match one left-most wildcard
+        # only match wildcard in left-most segment
```

Comment:
This comment still doesn't fell right, there is a wildcard _in_ the left-most segment. I think it should say something like "only match wildcards when they are the only thing in the left-most segment". What do you think?